docs/usage/getting-started/troubleshooting.md

```diff
@@ -0,0 +1,66 @@
+# Troubleshooting Renovate
+
+Learn how to troubleshoot problems with Renovate, where to find the logging output, and how to get help if needed.
+
+## Getting the logs
+
+Renovate logs the most recent jobs.
+Where you can find the logs depends on how you're running Renovate.
+
+### GitHub Hosted app
+
+Each pull request from Renovate contains a link to the WhiteSource Renovate Dashboard in the PR body text.
+
+The text you're looking for is:
+
+> This PR has been generated by WhiteSource Renovate. View repository job log here.
+
+Click on the blue text "here" to go to the WhiteSource Renovate Dashboard.
+Sign in with your GitHub or GitLab account.
+Once you're logged in, you can see the logs for the Renovate jobs on your repository.
+
+Renovate only keeps the logs for the last 3 days.
+
+When you've clicked on a recent job, you'll be able to select a debug level that you care about.
+For a full overview, make sure you select the DEBUG log level.
+
+### Selfhosted
+
+TODO: Explain log directory structure. Will probably be different depending on self-hosted platform...
+
+TODO: Explain how long logs are kept by default on differnet platforms.
+
+TODO: Explain how to toggle log level: <https://docs.renovatebot.com/examples/self-hosting/#about-the-log-level-numbers>
+
+## Log debug levels
+
+There are different severity levels for the log output.
+From least severe to most severe:
+
+- DEBUG
+- INFO
+- WARN
+- ERROR
+- FATAL
+
+As you can probably imagine, the stuff that's really going wrong will likely have a level of WARN or higher.
+
+TODO: What do the Renovate maintainers recommend as debug level?
```

Comment:
I'm not a heavy user of the logs, Renovate bot usually does what I want anyways.
So I'm not familiar with the workflow that you as maintainers have when troubleshooting.